atomicapp/nulecule/lib.py

```diff
@@ -47,12 +78,25 @@ def merge_config(self, to_config, from_config):
                     to_config[group][key] = value
 
     def get_context(self):
+        """
+        Get context data from config data for rendering an artifact.
+        """
         context = {}
         context.update(self.config.get('general') or {})
         context.update(self.config.get(self.namespace) or {})
         return context
 
     def get_provider(self, provider_key=None, dry=False):
+        """
+        Get provider key and provider instance.
+
+        Args:
+            provider_key (str or None): Name of provider
+            dry (bool): Do not make change to the host system when True.
```

Comment:
`Do not make changes to the host system while True` also remove period end of sentence since all other args dont have it